### PR TITLE
add MisterFusion by @szk-atmosphere

### PR DESCRIPTION
**Pull request by szk-atmosphere**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -303,7 +304,7 @@ Projects in Swift language will be marked with :large_orange_diamond: and :watch
  * [Route-me](https://github.com/route-me/route-me) - Open source map library for iOS.
  * [NAMapKit](https://github.com/neilang/NAMapKit) - Allows you to use custom maps in iphone applications and attempts to mimics some of the behaviour of the MapKit framework.
  * [Mapbox GL](https://github.com/mapbox/mapbox-gl-native) - An OpenGL renderer for Mapbox Vector Tiles with SDK bindings for iOS.
- * [CMMapLauncher](https://github.com/citymapper/CMMapLauncher) - iOS library that makes it quick and easy to show directions in various mapping applications.
```
Comment (szk-atmosphere) [inline README.md]: changes in this line seems like different character code

File: README.md [hunk 2]
```diff
@@ -303,7 +304,7 @@ Projects in Swift language will be marked with :large_orange_diamond: and :watch
  * [Route-me](https://github.com/route-me/route-me) - Open source map library for iOS.
  * [NAMapKit](https://github.com/neilang/NAMapKit) - Allows you to use custom maps in iphone applications and attempts to mimics some of the behaviour of the MapKit framework.
  * [Mapbox GL](https://github.com/mapbox/mapbox-gl-native) - An OpenGL renderer for Mapbox Vector Tiles with SDK bindings for iOS.
- * [CMMapLauncher](https://github.com/citymapper/CMMapLauncher) - iOS library that makes it quick and easy to show directions in various mapping applications. 
+ * [CMMapLauncher](https://github.com/citymapper/CMMapLauncher) - iOS library that makes it quick and easy to show directions in various mapping applications.
```
Comment (szk-atmosphere) [inline README.md]: changes in this line seems like different character code

Comment (dkhamsing) [inline README.md]: ok

**Comment by dkhamsing** [comment]
Thanks

**Comment by szk-atmosphere** [comment]
Thank you!

**Pull request merged**